values/jobs/autoconfig.gotmpl

```diff
@@ -0,0 +1,36 @@
+{{- $v := .Values }}
+{{- $c := $v.charts }}
+{{- $k := $c.keycloak }}
+type: Job
+name: autoconfig
+
+image:
+  repository: docker.io/otomi/otomi-e2e
+  tag: autoconfig
+  pullPolicy: Always
+
+script: | 
+    {{ if eq $v.otomi.hasExternalIDP false }}
+    echo "Creating Keycloak user"
+    yarn cypress run --record false --env BASE_URL={{$v.cluster.domainSuffix}} --spec 'cypress/integration/otomi-applications/10-keycloak.feature'
+    {{ end }}
+    {{ if eq $c.gitea.enabled true }}
+    echo "Activate drone"
+    yarn cypress run --record false --env BASE_URL={{$v.cluster.domainSuffix}} --spec 'cypress/integration/otomi-applications/20-drone.feature'
+    {{ end }}
+
+nativeSecrets:
+  CYPRESS_ROOT_USERNAME: {{$k.adminUsername}}
+  CYPRESS_ROOT_PASSWORD: {{$k.adminPassword}}
+  CYPRESS_ADMIN_USERNAME: 'admin'
```

Comment:
I talked with Sander and it is better to create `otomi-admin` user
Then each application (including gitea) will have the same user name

Reply:
Done :)